ui/pages/confirmations/components/snap-account-error-message/snap-account-error-message.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import ActionableMessage from '../../../../components/ui/actionable-message';
+import { Text } from '../../../../components/component-library';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const SnapAccountErrorMessage = ({
+  message,
+  learnMoreLink,
+  error,
+}: {
+  message: string;
+  learnMoreLink?: string;
+  error?: string;
+}) => {
+  const t = useI18nContext();
+
+  return (
+    <React.Fragment>
```

Comment:
This is not and issue but more of a nice to know. You can create a fragment with the following syntax...

```tsx
     <>
        {' '}
        <a href={learnMoreLink} rel="noopener noreferrer" target="_blank">
          {t('learnMoreUpperCase') as string}
        </a>
      </>
```

Reply:
ok, good to know, I'll change that! Thanks. :)